### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christie26**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # bubble-chat
 visitor interaction project during my architecture exhibition
+
+notion page : https://www.notion.so/Exhibition-Interaction-Project-23-02-23-06-28d66e5152134bda83007e629b749905?pvs=4
```
Comment (wendrul) [inline README.md]: It would look better as a markdown hyperlink, i.e

```
[Notion Page](notion_url)
```

**Review by wendrul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (christie26) [reply README.md]: thank you 

> you're kind
`[Notion page]`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # bubble-chat
 visitor interaction project during my architecture exhibition
 
-notion page : https://www.notion.so/Exhibition-Interaction-Project-23-02-23-06-28d66e5152134bda83007e629b749905?pvs=4
+[Notion page] (https://www.notion.so/Exhibition-Interaction-Project-23-02-23-06-28d66e5152134bda83007e629b749905?pvs=4)
```
Comment (wendrul) [inline README.md]: This space is breaking the rendering, please remove it

**Review by wendrul — CHANGES_REQUESTED** [review]
please update this

**Review by wendrul — APPROVED** [review]
LGTM!

**Pull request merged**